spec/index.html

```diff
@@ -523,8 +524,8 @@ <h3>Languages: <code>xml:lang</code></h3>
     to allow the identification of content language.
     The <code>xml:lang</code> attribute can be used on any node element or property element
     to indicate that the included content is in the given language.
-    <a href="#section-Syntax-datatyped-literals">Typed literals</a>
-    which includes <a href="#section-Syntax-XML-literals">XML literals</a>
+    <a href="#section-Syntax-datatyped-literals" class="sectionRef"></a>
+    which includes <a href="#section-Syntax-XML-literals" class="sectionRef"></a>
```

Comment:
This  change should IMO be reverted, because it makes the sentence meaningless, replacing nouns ("typed literals", "XML literals") with section titles ("2.9 Typed Literals: `rdf:datatype`", "2.8 XML Literals: `rdf:parseType="Literal"`").

Reply:
Good point.